arbitrum-client/Cargo.toml

```diff
@@ -3,6 +3,9 @@ name = "arbitrum-client"
 version = "0.1.0"
 edition = "2021"
 
+[package.metadata]
+integration_uses_devnet = true
```

Comment:
can we do:
```toml
[package.metadata.integration]
uses_devnet: true
```
We will likely put more keys under this as the integration test functionality builds out